pkg/model/log.go

```diff
@@ -4,6 +4,21 @@ import (
 	"context"
 )
 
+const (
+	METRIC_LOG_CREATE_METADATA = "log_create_metadata"
```

Comment:
Usually Go constants just use the standard `PascalCase` (e.g., `MetricLogCreateMetadata`). I think this is fine if you want to leave it, though, although it wouldn't surprise me if we find a linter that wants it changed in the future.

Reply:
Yeah, I follow that convention elsewhere. Just an inadvertent slip back to Java convention; I did not even notice I did that ...
I will update accordingly.